js/api.js

```diff
@@ -1,10 +1,16 @@
 function commonAPI(url, action, param){
```

Comment:
0x5Bb3dbb6408C011e82DED83b236e1943c77c6600

Reply:
Will it connect the api endpoints to this from multiple sites automatically